docs/content/src/tutorial-get-the-code.md

```diff
@@ -0,0 +1,7 @@
+## Get the code
+
+Download the ZIP file and extract the contents to get the code.
+
+<a href="../../../../../assets/zip/<<<MD_FILE_NAME>>>.zip">
+    <img src="../../../../../assets/img/download-zip.png" width="200" alt="Download ZIP">
+</a>
```

Comment:
Do we put the `ballerina pull` command as well. In that case will it work with `ballerina pull -t wso2/<<<MD_FILE_NAME>>>`